.eslintrc

```diff
@@ -36,6 +36,7 @@
     "@typescript-eslint/no-unused-vars": ["error", {}],
     "@typescript-eslint/no-explicit-any": "off",
     "no-useless-escape": "off",
-    "react/no-children-prop": "off"
+    "react/no-children-prop": "off",
+    "react/prop-types": "off"
```

Comment:
Why is react/prop-types off here? Seems like it'll hide problems that should be fixed.

Reply:
It was requiring prop types in implementations as well as definitions. Typescript already covers this.